frontend/templates/views/partials/footer.j2

```diff
@@ -68,7 +68,7 @@
       <div class="ap-o-footer-nav-column">
         <h5 class="ap-o-footer-nav-title">{{_('Events')}}</h5>
         <ul class="ap-o-footer-nav-menu">
-          <li class="ap-o-footer-nav-item"><a href="{{ g.doc('/content/amp-dev/events/amp-conf-2019.html', locale=doc.locale).url.path }}" class="ap-o-footer-nav-link"><div class="ap-a-ico ap-o-footer-nav-icon"><svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#internal"></use></svg></div>{{_('AMP Conf 2019')}}</a></li>
+          <li class="ap-o-footer-nav-item"><a href="{{ g.doc('https://blog.amp.dev/2020/02/20/amp-conf-2020-return-to-nyc/', locale=doc.locale).url.path }}" class="ap-o-footer-nav-link"><div class="ap-a-ico ap-o-footer-nav-icon"><svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#internal"></use></svg></div>{{_('AMP Conf 2020')}}</a></li>
```

Comment:
Careful: this is an absolute link and shouldn't be enclosed in `g.doc()`. Wondering why this isn't throwing problems for anyone else. For me it breaks for all pages with:

```
An error occurred building <RenderDocumentController(/content/amp-dev/about/success-stories/findomestic.yaml)>
"https://blog.amp.dev/2020/02/20/amp-conf-2020-return-to-nyc/" is not a path to a document.
```